test/git_resolver_test.cr

```diff
@@ -8,6 +8,11 @@ module Shards
 
       create_git_repository "library", "0.0.1", "0.1.0", "0.1.1", "0.1.2"
       create_git_release "library", "0.2.0", shard: "name: library\nversion: 0.2.0\nauthors:\n  - julien <julien@portalier.com>"
+
+      Dir.cd(git_path("library")) do
+        # Create a version tag not prefixed by 'v' which should be ignored
+        run "git tag 99.9.9"
+      end
```

Comment:
Just one tiny thing: could you add a `create_git_tag(project, tag)` helper in `test/support/factories.cr`? then could it `create_git_tag "library", "99.9.9"`?

That would be perfect :)